vscode/microsoft-kiota/src/steps.ts

```diff
@@ -287,32 +291,36 @@ export async function generateSteps(existingConfiguration: Partial<GenerateState
             state.pluginName = await input.showInputBox({
                 title: `${l10n.t('Create a new plugin')} - ${l10n.t('plugin name')}`,
                 step: step++,
-                totalSteps: 4,
+                totalSteps: 3,
                 value: state.pluginName ?? '',
                 placeholder: 'MyPlugin',
                 prompt: l10n.t('Choose a name for the plugin'),
                 validate: validateIsNotEmpty,
                 shouldResume: shouldResume
             });
         }
+        state.pluginTypes = ['ApiPlugin'];
         updateWorkspaceFolder(state.pluginName);
-        return (input: MultiStepInput) => inputPluginType(input, state);
+        return (input: MultiStepInput) => inputPluginOutputPath(input, state);
     }
-    async function inputPluginType(input: MultiStepInput, state: Partial<GenerateState>) {
+    async function chooseOtherGenerationType(input: MultiStepInput, state: Partial<GenerateState>) {
         if (!isDeepLinkPluginTypeProvided) {
-            const items = ['API Plugin', 'Open AI'].map(x => ({ label: x }) as QuickPickItem);
+            const items = ['API Manifest', 'Open AI Plugin'].map(x => ({ label: x }) as QuickPickItem);
             const pluginTypes = await input.showQuickPick({
-                title: l10n.t('Choose a plugin type'),
+                title: l10n.t('Choose a type'),
                 step: step++,
                 totalSteps: 4,
                 placeholder: l10n.t('Select an option'),
                 items: items,
                 validate: validateIsNotEmpty,
                 shouldResume: shouldResume
             });
-            pluginTypes.label === 'API Plugin' ? state.pluginTypes = ['ApiPlugin'] : state.pluginTypes = ['OpenAI'];
+            state.pluginTypes = pluginTypes;
```

Comment:
```suggestion
            pluginTypes.label === 'API Manifest' ? state.pluginTypes = ['ApiManifest'] : state.pluginTypes = ['OpenAI'];
            
```

Reply:
Otherwise this would result to state.pluginTypes to be an object e.g pluginTypes =
{label: 'Open AI Plugin'} instead of an array which is not an acceptable plugin type.